editor/src/components/inspector/common/css-utils.ts

```diff
@@ -943,11 +943,35 @@ export function parseGridRange(
   }
 }
 
+// this regex matches 'repeat ( [unit] , [value] )', capturing groups around the unit (1) and the value (2)
+const reRepeatFunction = /repeat\s*\(\s*(\d+)\s*,\s*([^,\)]+)\s*\)/
```

Comment:
just a note - maybe add a TODO to support non-units repeaters (i.e `auto-fit`, `auto-fill`, etc)?

Reply:
makes sense, done ✅